scripts/qa_crowbarsetup.sh

```diff
@@ -3133,6 +3134,20 @@ function set_proposalvars
     fi
 }
 
+function set_noproxyvar
+{
+    [[ $http_proxy ]] || [[ $https_proxy ]] || return
+    no_proxy="$adminip,$no_proxy"
+    export no_proxy="${no_proxy%,}";
```

Comment:
If I understand bash correctly, `export` just sets a bit and it does not matter where we do that.
Also, net_public might not be set here, so we would not always reach the later part